src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final long ID = 123L;
+    private static final int AGE = 18;
+    private static final String LOGIN = "firstuser@mail.com";
+    private static final String PASSWORD = "askj435%";
+    private static final User EMPTY_USER = new User();
+    private static final User CUSTOM_USER = new User();
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    @DisplayName("Set all values")
+    static void createUsers() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registerSingle_ValidData_Ok() {
+        StorageDao storageDao = new StorageDaoImpl();
+        User validUser = new User(ID, LOGIN, PASSWORD, AGE);
```

Comment:
Positive edge cases must be tested
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/bce79c33-4260-47f1-a3a5-6b8ff9ce10a3)